tapeforms/utils.py

```diff
@@ -0,0 +1,11 @@
+from itertools import chain
+
+
+def join_css_class(css_class, *args, as_set=False):
```

Comment:
I don't see how as_set is used and would like to rename args to something more clear.

```suggestion
def join_css_class(css_class, *additional_css_classes):
```

Reply:
Good catch, I forgot to remove `as_set` which was finally not used. I have renamed `args` to `additional_css_classes` as suggested, which is more clear indeed.